src/config/version.py

```diff
@@ -51,7 +51,9 @@ def is_prerelease_version(version):
 
 def read_release_version():
     try:
-        f = open("RELEASE-VERSION", "r")
+        import util
```

Comment:
Because this imports util and we happen to use this file as a script (This itself is probably bad practice), the appveyor build has lost its version number: https://ci.appveyor.com/project/Sheeo/client/build/392#L147